packages/MusicNotation-Scripting.package/MNNoteBuilder.class/instance/^plus.st

```diff
@@ -0,0 +1,6 @@
+arithmetic
++ aNoteBuilder
+	| pitches |
+	pitches := self pitches copy.
+	pitches addAll: aNoteBuilder pitches.
+	^ MNNoteBuilder context: self context pitches: pitches
```

Comment:
Hier sollte noch überprüft werden, ob `self context = aNoteBuilder context` gilt, ansonsten sollte ein `MNInvalidValue` Error geworfen werden.

Reply:
Use `self class` instead of `MNNoteBuilder`:
```squeak
self class context: "..." pitches: "..."
```